### ISSUE TITLE
[BUG] Using Langchain Async Support on the dashboard will display two pieces of data.

### ISSUE THREAD

**Issue by zongyicoding** [truera/trulens#1370]
**Bug Description**
What happened?
- Hi Trulens teams, I encountered the a question of receiving duplicate responses while using Trulens Async Support.
[link here](https://www.trulens.org/trulens_eval/tracking/instrumentation/langchain/)

**To Reproduce**
Which steps should someone take to run into the same error? A small, reproducible code example is useful here.
```jupyter-notebook
from langchain import LLMChain
from langchain.callbacks import AsyncIteratorCallbackHandler
from langchain.chains import LLMChain
from langchain.prompts import PromptTemplate
from langchain_openai import ChatOpenAI

from trulens_eval import TruChain

# Set up an async callback.
callback = AsyncIteratorCallbackHandler()

# Setup a simple question/answer chain with streaming ChatOpenAI.
prompt = PromptTemplate.from_template("Honestly answer this question: {question}.")
llm = ChatOpenAI(
    temperature=0.0,
    streaming=True, # important
    callbacks=[callback]
)
async_chain = LLMChain(llm=llm, prompt=prompt)
async_tc_recorder = TruChain(async_chain)

with async_tc_recorder as recording:
    await async_chain.ainvoke(input=dict(question="What is 1+2? Explain your answer."))
``` 
**Expected behavior**
- When asking a question, only get one response.

**Relevant Logs/Tracebacks**
Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks. If the issue is related to the TruLens dashboard, please also include a screenshot.

**Environment:**

- OS: [e.g. MacOS, Windows] MacOS
- Python Version 3.12
- TruLens version 0.33.0
- Versions of other relevant installed libraries

**Additional context**

**Comment by sfc-gh-jreini** [comment]
Hi @zongyicoding - @sfc-gh-pmardziel is working on a fix

**Comment by sfc-gh-jreini** [comment]
Hey @zongyicoding - we have a fixes coming for async and streaming in [this PR](https://github.com/truera/trulens/pull/1441).

Feel free to install it from the branch if you'd like to try it now, otherwise we'll have it in the next release.

**Comment by sfc-gh-jreini** [comment]
Hey @zongyicoding - fixes for this were released in [TruLens 1.1](https://github.com/truera/trulens/releases/tag/trulens-1.1.0). Please give it a try and let us know if you have any issues.

```
pip install -U trulens-core trulens-apps-langchain
```
